common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -198,6 +198,10 @@ def send(self, signal_name, **kwargs):
             log.info('Sent %s signal to %s with kwargs %s. Response was: %s', signal_name, receiver, kwargs, response)
 
 
+# to allow easy imports
+globals().update({sig.name.upper(): sig for sig in SignalHandler.all_signals()})
```

Comment:
Please don't munge the globals. It can confuse IDEs and make it harder to understand what's going on later.

Reply:
I don't like it either, but there's a feature of Pluggable Apps where you can [specify a signal in your apps.py](https://github.com/edx/edx-when/blob/master/edx_when/apps.py#L30). That doesn't work for `SignalHandler.course_published` because it's not in the module scope. Most other signals are not class attributes. 

Do you have another suggestion?

Reply:
Argh, I didn't realize that. Okay, fine to keep this then. I'll take it as a todo item to update `plugin_signals.py` to do the right thing later.